cylc/flow/scheduler.py

```diff
@@ -555,8 +555,22 @@ async def configure(self, params):
                     timer.reset()
                 self.timers[event] = timer
 
-        if self.is_restart and not self.pool.get_tasks():
-            # This workflow completed before restart; wait for intervention.
+        if self.is_restart and (
+            # workflow has completed
+            not self.pool.get_tasks()
+            # workflow has hit the "stop after cycle point"
```

Comment:
Suggest that this should be documented in `globalcfg.py` since this setting now applies to stopped as well as completed workflows.

Reply:
Under what config?